common/src/main/java/org/apache/carbondata/common/exceptions/sql/MalformedIndexCommandException.java

```diff
@@ -21,21 +21,21 @@
 import org.apache.carbondata.common.annotations.InterfaceStability;
 
 /**
- * This exception will be thrown when Datamap related SQL statement is invalid
+ * This exception will be thrown when index related SQL statement is invalid
```

Comment:
@niuge01 and @jackylk : As per my understanding. After this PR. There won't be datamap word in the code. It is either index or materialized view. But **many package names still has datamap** [exmaple CgDatamap, FgDatamap] and files also exist with datamap name. [Example DataMapStatus, DataMapStatusManager] 
Do we need to completly remove if weare doing this change.